bridge-ffi/src/lib.rs

```diff
@@ -0,0 +1,833 @@
+#![allow(
+    // Non-camel-case types needed since this is exported as a C header and we
+    // want C-like underscores in our type names
+    non_camel_case_types,
+
+    // We choose to have narrow "unsafe" blocks instead of marking entire
+    // functions as unsafe. Even the example in clippy's docs at
+    // https://rust-lang.github.io/rust-clippy/master/index.html#not_unsafe_ptr_arg_deref
+    // cause a rustc warning for unnecessary inner-unsafe when marked on fn.
+    // This check only applies to "pub" functions which are all exposed via C
+    // API.
+    clippy::not_unsafe_ptr_arg_deref,
+)]
+
+mod wrappers;
+
+use prost::Message;
+use temporal_sdk_core_protos::coresdk::bridge;
+
+extern crate libc;
```

Comment:
I don't think this extern declaration is necessary -- if it is style-wise I generally prefer these at the very top, but they should only be necessary with `#[macro_use]`

Reply:
Removed